backend/WebApi/appsettings.Development.json

```diff
@@ -5,5 +5,11 @@
       "Microsoft": "Warning",
       "Microsoft.Hosting.Lifetime": "Information"
     }
+  },
+  "EmailOptions": {
```

Comment:
I think we should leave these fields as blank in our remote repo, change them to our real credentials in our local repos and then add this file to `.gitignore`. Same with `appsettings.json`

Reply:
We updated the way of working with this credentials by using the secret manager. Right now there is no sensitive information in any file. Maybe we should delete something in the git repository because our credentials have been exposed? @audrius-savickas